tests/baselines/reference/implicitAnyInCatch.types

```diff
@@ -1,5 +1,8 @@
 //// [tests/cases/compiler/implicitAnyInCatch.ts] ////
 
+=== Performance Stats ===
+Type Count: 1,000
```

Comment:
Wow, how are we instantiating this many types in a test that doesn't do anything? I guess it's because of `var key in this`?

Reply:
the instantiable additions are probably just enough to bump us over the limit for logging the type count.